frontend/src/metabase/admin/tools/components/ToolsUpsell/ToolsUpsell.tsx

```diff
@@ -0,0 +1,10 @@
+import { UpsellPerformanceTools } from "metabase/admin/upsells";
+import { Center } from "metabase/ui";
+
+export const ToolsUpsell = () => {
+  return (
+    <Center inline w="100%">
+      <UpsellPerformanceTools source="settings-tools" />
+    </Center>
+  );
+};
```

Comment:
Not sure if this was the right approach, tbh.
I could achieve two things by creating this light wrapper:

1. Control the layout and positioning of the upsell component without having to hard code the position into `UpsellPerformanceTools.tsx`
2. Pass the correct `source` prop without having to do it in the `routes.jsx` (which keeps routes a bit cleaner)